Makefile

```diff
@@ -0,0 +1,84 @@
+# Makefile for the charm store client.
+
+ifndef GOPATH
```

Comment:
I'm not super familiar with ifndef and how it handles vars vs everywhere else. Seems there's as many different ways as there are tools that interact.